daemon/cmd/daemon_main.go

```diff
@@ -1433,6 +1445,24 @@ func runDaemon() {
 	iptablesManager := &iptables.IptablesManager{}
 	iptablesManager.Init()
 
+	var wgAgent *wireguard.Agent
+	if option.Config.EnableWireguard {
```

Comment:
Somewhere (maybe not here), we should check that IPsec and Wireguard are mutually exclusive.

Reply:
Also, do we require to have remote node identities (given the earlier use of `REMOTE_NODE_ID`)?

Reply:
not required, if the identity is seen in the ipcache then the code handles it. We *might* be able to optimize away that check but we'd have to look closely at the details. Unless we're really itching to minimize instructions, I would avoid looking into such an optimization for now.